tests/kafkatest/tests/connect/templates/connect-distributed.properties

```diff
@@ -19,6 +19,10 @@ bootstrap.servers={{ kafka.bootstrap_servers(kafka.security_config.security_prot
 {{ kafka.security_config.client_config().props("consumer.") }}
 
 group.id={{ group|default("connect-cluster") }}
+client.id={{ node.account.hostname }}
```

Comment:
Unfortunately, besides the `client.id` in the worker, this will also set the `client.id` in all the internal Kafka clients (producer, consumer, admin client) deployed for handling internal topics, for instance by KafkaBasedLog. However, improving readability for assignments, seems preferable than distinguishing between clients for internal topics. WDYT?

Reply:
I'm a bit worried about the system tests now always setting a custom `client.id` now on all internal producers and consumers, especially if this is not as typical with users as that would mean the system tests are not really representative of normal Connect usage. And considering that [KIP-411](https://cwiki.apache.org/confluence/display/KAFKA/KIP-411%3A+Make+default+Kafka+Connect+worker+task+client+IDs+distinct) recently changed the producer and consumer client IDs when `client.id` is *not* set, making this change means that Connect system tests will never exercise that behavior.

Reply:
ok, never mind then. I'll remove this commit. Renaming the ids on the failure logs after the fact could be another option to make worker ids more readable.